src/Whim/Template/whim.config.csx

```diff
@@ -113,7 +113,8 @@ void DoConfig(IContext context)
 			(id) => SliceLayouts.CreatePrimaryStackLayout(context, sliceLayoutPlugin, id),
 			(id) => SliceLayouts.CreateSecondaryPrimaryLayout(context, sliceLayoutPlugin, id),
 			(id) => new FocusLayoutEngine(id),
-			(id) => new TreeLayoutEngine(context, treeLayoutPlugin, id)
+			(id) => new TreeLayoutEngine(context, treeLayoutPlugin, id),
+			(id) => new FreeLayoutEngine(context, id)
 		};
 }
```

Comment:
To see what we want to do:

<details>
<summary>`FocusIndicator` can behave strangely sometimes while moving or resizing windows (click me for images)</summary>

![image](https://github.com/dalyIsaac/Whim/assets/2392459/bbffb097-1fdc-4032-a216-cb85ba791d68)
![image](https://github.com/dalyIsaac/Whim/assets/2392459/6bcccc08-7867-4cdb-a02c-03d74e8fadf8)

</details>

Reply:
Yeah that's not ideal. It'd probably be easier to defer this to https://github.com/dalyIsaac/Whim/issues/669.

Reply:
Good for me